10_fibonacci/solution/fibonacci-solution.js

```diff
@@ -1,6 +1,6 @@
 const fibonacci = function(count) {
     if (count < 0) return "OOPS";
-    if (count === 0) return 0;
+    if (count == 0) return 0;
```

Comment:
I don't like this, we should be explicit. Let's convert count to a number first.

Reply:
Thanks, adapting it now.